rust/numaflow-models/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2021"
 
 [dependencies]
 k8s-openapi = { version = "0.22.0", features = ["v1_29"] }
-kube = "0.93.1"
+kube = "0.94.0"
```

Comment:
any particular reason? we have marked `k8s-openapi` to include only v1_29 feature. we might have to pin `k8s-openapi` too.

Reply:
To avoid unexpected new release caused build problem